.gitlab-ci.yml

```diff
@@ -1,48 +1,63 @@
 variables:
   GIT_SUBMODULE_STRATEGY: recursive
 
-cache:
-  key: ${CI_COMMIT_REF_SLUG}
-  paths:
-    - node_modules/
-    - dist/
-    - proto/
-    - docs/
-
 stages:
   - check
   - build
   - deploy
 
-before_script:
-  - node --version
-  - npm install
-
-prettier:
+lint:
+  image: registry.gitlab.com/matrixai/engineering/maintenance/gitlab-runner
   stage: check
-  image: node:12
   script:
-    - npm run lint
+    - nix-shell --packages nodejs --run "npm install"
+    - nix-shell --packages nodejs --run "npm run lint"
 
-jest:
-  stage: check
+test-npm:
   image: node:12
+  stage: check
   script:
+    - npm install
     - npm run test
 
-build_all:
+# # This requires QEMU to work on our image which currently does not work
```

Comment:
We can just remove this, QEMU testing shouldn't be done as part of CI/CD here. The CI/CD right now can only do unit-level CI/CD. Once we have Matrix OS ready, then we can do this more automated.

Reply:
I have removed this part of the file